x/campaign/keeper/mainnet_account_test.go

```diff
@@ -0,0 +1,60 @@
+package keeper_test
+
+import (
+	"strconv"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	keepertest "github.com/tendermint/spn/testutil/keeper"
+	"github.com/tendermint/spn/x/campaign/keeper"
+	"github.com/tendermint/spn/x/campaign/types"
+)
+
+// Prevent strconv unused error
+var _ = strconv.IntSize
+
+func createNMainnetAccount(keeper *keeper.Keeper, ctx sdk.Context, n int) []types.MainnetAccount {
+	items := make([]types.MainnetAccount, n)
+	for i := range items {
+		items[i].CampaignID = uint64(i)
+		items[i].Address = strconv.Itoa(i)
```

Comment:
We should test it with a real address like with `sample.AccAddress()`

One issue if we do this is that the `TestMainnetAccountQueryPaginated` above will fail because the order of the elements in the store will not be predictable.
I'm personally in the opinion that `TestMainnetAccountQueryPaginated` can be removed for now if it is problematic. Test for the `cli` package are not the most important priority

Reply:
Well for `TestMainnetAccountQueryPaginated ` we can in fact just check if each paginated result is a subset of the whole testcase array

Reply:
Great, I made the changes here. Also, I check if the result count is less or equal to the step

https://github.com/tendermint/spn/pull/298/commits/0344a0b7faaee8b1dd0dc7c737679b4aebb2e2e0